cypress.config.js

```diff
@@ -63,7 +65,8 @@ module.exports = defineConfig({
     supportFile: "cypress/support/e2e.ts",
     viewportHeight: 1000,
     viewportWidth: 1280,
-    experimentalSessionAndOrigin: !!process.env.REACT_APP_AUTH0_CLIENTID,
+    experimentalSessionAndOrigin:
```

Comment:
can we branch of the v12.0.0 branch so we can remove this config value?

Reply:
I can likely take care of that tomorrow to create a `v12.0.0` of realworld app and base the cognito and okta PRs off of it.

Reply:
@emilyrohrbough any idea what the easiest way to do this is? we usually bump this when we release? Just cut a branch and install a prerelease binary?

Reply:
We plan to be in dev by the end of the day, so you could just update this monday.

Reply:
Ill make a note so I dont forget!